src/main/java/nu/studer/gradle/buildscan/teamcity/TeamCityBuildScanPlugin.java

```diff
@@ -22,25 +22,23 @@ public void apply(Project project) {
         if (System.getenv(TEAMCITY_VERSION_ENV) != null && System.getenv(GRADLE_BUILDSCAN_TEAMCITY_PLUGIN_ENV) == null) {
             project.getLogger().quiet(ServiceMessage.of(BUILD_SCAN_SERVICE_MESSAGE_NAME, BUILD_SCAN_SERVICE_STARTED_MESSAGE_ARGUMENT).toString());
 
-            project.getPluginManager().withPlugin(BUILD_SCAN_PLUGIN_ID, appliedPlugin -> {
```

Comment:
By removing this, the code has become order-depenend with regard to how the plugins are applied.

Reply:
True, has been fixed.